README.rst

```diff
@@ -35,6 +36,8 @@ wok builds the site by assembling several components:
 
 To build the site, run ``make build``.  This will pull together all the componenents into a set of HTML files in ``output/``.
 
+Windows users: you need to run the (extensionless) ``wok`` script in c:/pythonxx/scripts. eg ``py -2 c:\python27\scripts\wok``.
```

Comment:
@tjguk -- can you check whether running ``wok --serve`` works on Windows?  If it does, could you add a note about that to the line below?